index.html

```diff
@@ -0,0 +1,198 @@
+        <div class="tablet:grid-col-6 padding-y-3 tablet:display-flex">
+          <div>{% include sprite.html icon="identification" size="7" %}</div>
+          <div class="tablet:margin-left-2">
+            <h3 class="margin-y-1">Receive standardized claims data</h3>
+            <p>
+              Get claims data that follows the
+              <a href="https://hl7.org/fhir/R4/overview.html">HL7 Healthcare Interoperability Standard</a>
+              required by Centers for Medicare & Medicaid Services (CMS).
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <!--
+    -->
+    <div class="grid-row grid-gap-4 desktop:grid-gap-6 padding-y-7">
+      <div class="tablet:order-last tablet:grid-col display-flex flex-align-center tablet:flex-justify-center">
+        <img class="width-auto height-full" style="object-fit: contain;"
+          src="{{ '/assets/img/understanding-the-data.svg' | relative_url }}"
+          alt="Illustration of two people collaborating using a laptop and discussing data, charts, and communication." />
+      </div>
+      <div class="tablet:grid-col padding-top-4 tablet:padding-top-0 display-flex flex-align-center">
+        <div>
+          <h2>Using the data</h2>
+          <p>BCDA is an Application Programming Interface (API) that updates enrollee claims data at least once a week.
+            Data is provided in NDJSON format and includes:</p>
+          <ul>
+            <li>Medicare enrollee identifiers</li>
+            <li>diagnosis codes</li>
+            <li>dates and times of service</li>
+          </ul>
+          <p>
+            <a href="{{ '/bcda-data.html' | relative_url }}" class="usa-button usa-button--unstyled">Learn about the
+              data {% include sprite.html icon="arrow_forward" %}</a>
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div>
+    <div class="grid-row">
+      <div class="bg-primary grid-col-12 desktop:grid-col-6">
+        <div style="max-width: calc(75rem / 2);"
+          class="desktop:margin-left-auto padding-x-3 desktop:padding-right-6 height-full padding-y-8 text-white display-flex flex-column">
+          <div class="border-bottom border-white">
+            <h2 class="margin-bottom-2">Featured</h2>
+          </div>
+          <h3 class="font-sans-lg" style="line-height: 1.4;">Get claims data sooner with partially adjudicated
+            claims</h3>
+          <div>
+            <p class="margin-bottom-0">Through BCDA, health care organizations can access claims 2 to 4 days after
+              they’re submitted instead
+              of 14 or more. These “partially adjudicated” claims have valuable data but haven’t been fully
+              processed and paid by Medicare systems.</p>
+          </div>
+          <div class="margin-top-auto">
+            <div class="margin-top-5">
+              <a href="#" class="usa-button usa-button--outline usa-button--inverse">
+                Partially adjudicated data
```

Comment:
```suggestion
                Learn more about partially adjudicated claims data
```
Buttons should include a verb